src/dc_receive_imf.rs

```diff
@@ -237,6 +237,16 @@ pub(crate) async fn dc_receive_imf_inner(
         };
     }
 
+    if let Err(err) = contact::set_status(
```

Comment:
this is called even if status is `None` to allow removing the status, right? i thought first, an `if let some()` as above for the avatars is missing.

maybe add a little comment - otoh, this seems to be tested explictily in python :)

Reply:
Added the comment